src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -6,4 +6,6 @@
 public interface BookRepository extends JpaRepository<Book, Long> {
 
     Book findByAuthor(String author);
+
+    Book findByTitle(String title);
```

Comment:
I added this method because the CRUD had a GET route that brings books by title, so I added the method to avoid errors in the controller.

Reply:
No worries. This get's improved quite a lot in a later card, where you'll need to add filtering capabilities to the getAll resource through query parameters.